.pre-commit-config.yaml

```diff
@@ -1,17 +1,19 @@
 repos:
+  - repo: https://github.com/charliermarsh/ruff-pre-commit
```

Comment:
pre-commit isn't installed automatically, right?

Could pre-commit setup be added to the setup script? and/or a small dev section added to the readme on how to set it up?